backend/models.py

```diff
@@ -13,78 +12,83 @@ def __str__(self) -> str:
 
 
 class UploadRequest(models.Model):
-    user = models.ForeignKey(get_user_model(), on_delete=DO_NOTHING,
+    user = models.ForeignKey(get_user_model(), on_delete=models.DO_NOTHING,
                              null=True, blank=True)
 
 
 class FieldOfStudy(models.Model):
     DEGREE = (
-        ("7", "7 terms I"),
-        ("6", "6 terms I"),
-        ("3", "3 terms II"),
-        ("4", "4 terms II")
+        ("1", "Bachelor"),
+        ("2", "Master")
```

Comment:
Ja bym tu chyba dał polskie nazwy, bo po ang chyba nie jest tak fajnie oddzielony licencjat od inżynierki

Reply:
nie zaznaczam, bo to nie ma dla nas żadnego znaczenia - tylko robimy statystyki, więc nie ma dla nas znaczenia czy ktoś na drugi stopień przyszedł po licencjacie czy po inżynierce, a ułatwia to filtrowanie